### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_ Jobs_Keyboard: Tooltip is not provided for “Tick” marks present in the table.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3615]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS1.1.1; #Non-text Content; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS1.1.1

**Repro Steps:**
1.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA application.
3.Tab till View Menu-->Tool Windows-->Azure Explorer and press enter.
4.Tab till AzureHD InsightsLink a cluster and press enter.
5.Enter all the fields and press ok.
6.Tab till the cluster that was linked and press enter.
7.Select “Jobs” link.
8.Verify whether Tooltip is provided for “Tick” marks present in the table or not.

**Actual Result:**
Tooltip is not provided for “Tick” marks present in the table.

**Expected Result:**
Tooltip should be provided for “Tick” marks present in the table.

**User Impact:**
Keyboard users will face difficulty in understanding what information the “Tick” marks are containing if Tooltip is not provided for them.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

[Bug_tool tip is not provided for Tick marks present in the table in Jobs page.pptx](https://github.com/microsoft/azure-tools-for-java/files/3646733/Bug_tool.tip.is.not.provided.for.Tick.marks.present.in.the.table.in.Jobs.page.pptx)

**Comment by t-rufang** [comment]
Note: Right click action for single line in job view table is not accessible by keyboard.
![image](https://user-images.githubusercontent.com/32627233/65662515-0f08d080-e067-11e9-9510-ab76c6b7b870.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749715
